cg/models/pipeline_qc_metrics/pipeline_qc_metrics.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class LaneMetric(BaseModel):
+    """Model for lane specific sequencing metrics."""
+
```

Comment:
I think a lot more information about the fastq QC can be added here, such as from the trimming like:
- passed_filter_reads
- low_quality_reads
- too_many_N_reads
- low_complexity_reads
- too_short_reads
- read1_mean_length (after trimming)
- read2_mean_length (after trimming)